test/unit/account/search_test.rb

```diff
@@ -65,6 +65,22 @@ class Account::SearchTest < ActiveSupport::TestCase
     assert_equal 'fo\/o', Account.search_ids('fo/o').query
   end
 
+  test 'search with query by email' do
+    ThinkingSphinx::Test.run do
+     provider = FactoryBot.create(:provider_account)
+     buyer = FactoryBot.create(:simple_buyer, :provider_account => provider)
+     buyer2 = FactoryBot.create(:simple_buyer, :provider_account => provider)
+     buyer.users << FactoryBot.build(:admin, :account => buyer, :username => "uname1", :email => "foo@bar.co.uk")
+     buyer2.users << FactoryBot.build(:admin, :account => buyer2, :username => "uname2", :email => "foo@bar.com")
+  
+     Indexer.run_after_index do
+      buyers = provider.buyer_accounts.scope_search(:query => 'foo@bar.co.uk')
+      assert_equal 1, buyers.count
+      assert_equal buyer, buyers.first
```

Comment:
```suggestion
      assert_equal [buyer], buyers
```

So you get the count and the content in 1 single line 😄

Reply:
If you apply what I mentioned above, this should be:
```ruby
expected_buyer = User.find_by!(email: 'foo@bar.co.example.com').account
assert_equal [expected_buyer], buyers 
```